docs/use/setup-wallet/index.md

```diff
@@ -169,11 +170,11 @@ The Centrifuge Ledger app does not support all transactions.
 4. Select APPROVE or REJECT by pressing down both buttons on your Ledger device. 
 ![](./images/ledger-docs-9.png)
 
-5. Verify the transaction on the portal in the upper right corner. View the transaction by navigating to Network > Explorer, under recent events.
+5. Verify the transaction on Polkadot.js in the upper right corner. View the transaction by navigating to Network > Explorer, under recent events.
 ![](./images/ledger-docs-10.png)
 
-### Use Ledger device with Centrifuge Chain Portal
-In case you want to use your Ledger device in the Centrifuge [portal](https://portal.chain.centrifuge.io) you will have to add the device through the PolkadotJS [Extension](https://polkadot.js.org/extension).
+### Use Ledger device via Polkadot.js
```

Comment:
We should remove ledger for now. It's not supported afaik @devin-black

Reply:
Ah good to know! I've put in a commit to remove the instructions. This also includes a note saying Ledger doesn't support Centrifuge Chain.